pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -242,6 +242,25 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 				continue
 			}
 
+			// Check whether container is present in state, there may be 3 reasons why it's not present:
+			// - container is in Be/Bu pod
```

Comment:
This assumes that we are using the static policy as implemented. Can we update this to say "container is using the default CPU set"?

Reply:
In this case, there's no good reason to call AddContainer again.

Reply:
shouldn't that be up to the policy to decide ? I need to think about that...

Reply:
:+1: